.claude/skills/README.md

```
@@ -0,0 +1,113 @@
+**Resources**:
+- [Prow Dashboard](https://prow.ci.openshift.org/)
+- [CI Search](https://github.com/openshift/ci-search)
+
+## Usage
+
+Skills are reusable workflows that combine multiple tools and knowledge to accomplish specific tasks.
+
+### Invoking Skills
+
+Skills can be referenced in Claude conversations:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)